packages/fluent-editor/src/modules/ai/types.ts

```
@@ -0,0 +1,18 @@
+export interface OperationMenuItem {
+  id: string;
```

ESLint

[error] Unexpected separator (;).

(style/member-delimiter-style)